scss/bootstrap/overrides/style/_thumbnail.scss

```diff
@@ -0,0 +1,75 @@
+.crm-mosaico-template-item {
+  background: $crm-background-light;
+  border: 2px solid $crm-white !important;
+  @include prefix(border-radius, 4px, moz webkit ms);
```

Comment:
According to http://caniuse.com/#search=border-radius
we don't need prefixes anymore for border-radius